statistics/boostrap.go

```diff
@@ -36,13 +36,17 @@ func initStatsMeta4Chunk(is infoschema.InfoSchema, tables statsCache, iter *chun
 			continue
 		}
 		tableInfo := table.Meta()
-		tbl := &Table{
-			TableID:     tableID,
+		newHistColl := HistColl{
+			TableID:     tableInfo.ID,
+			HaveTblID:   true,
```

Comment:
HasTableID

Reply:
When will this value be false?

Reply:
When the `histColl` is held in other plan. It's false.